src/chart/Chart.ts

```diff
@@ -1,4 +1,4 @@
-import { Record as Neo4jRecord } from '../neo4j-driver-lite/neo4j-lite-web.esm';
```

Comment:
You should be able to use the types imported from 'neo4j-driver'

Reply:
Hmmm... That was a commit miss !